### PR TITLE
Longreads

### PR DESCRIPTION
**Pull request by kaijli**
Added support for use with JGI long reads metaflye workflow. Updated parts are the new [choose_assembly.wdl](https://github.com/microbiomedata/metaAssembly/blob/longreads/choose_assembly.wdl) and additional outputs from long reads. Also waiting for make_info task update / mergefor the metaflye workflow [on gitlab](https://code.jgi.doe.gov/BFoster/jgi_meta_wdl/-/merge_requests/13) .

### CODE REVIEW

**Review by mflynn-lanl — APPROVED** [review]

**Review by aclum — APPROVED** [review]

File: make_interleaved_WDL/make_interleaved_reads.wdl [hunk 1]
```diff
@@ -21,24 +28,25 @@ workflow make_interleaved_reads {
...
+	input{
 	Array[File] input_files
 	String output_file = "interleaved.fastq.gz"
-	
+	String container
+	}
 	command <<<
-	    if file --mime -b ${input_files[0]} | grep gzip; then 
-			paste <(gunzip -c ${input_files[0]} | paste - - - -) <(gunzip -c ${input_files[1]} | paste - - - -) | tr '\t' '\n' | gzip -c > ${output_file}
+		if file --mime -b ~{input_files[0]} | grep gzip; then
```
Comment (aclum) [inline make_interleaved_WDL/make_interleaved_reads.wdl:37]: Please review all the tasks in the wdl to make sure pipeline is set if there is more than one bash executable so errors are caught by cromwell. 
command is 
'set -euo pipefail'

File: test_output/test_assembly.wdl [hunk 2]
```diff
@@ -1,46 +1,55 @@
...
 task prepare {
+  input{
    String container
    String ref_json
    String url
-   command{
-       wget -O "input.fastq.gz" ${url}
-       wget -O "ref_json.json" ${ref_json}
    }
+   command <<<
```
Comment (aclum) [inline test_output/test_assembly.wdl:49]: same here all tasks should have 'set -euo pipefail' if the command block contains more than on bash command.

**Review by aclum — CHANGES_REQUESTED** [review]
Please review wdls to make sure 'set -euo pipefail' is set in the command block to prevent silent failures.

File: test_output/test_assembly.wdl (hunk 2 above)
Comment (aclum) [reply test_output/test_assembly.wdl:49]: This command block still needs a pipefail.

**Review by aclum — APPROVED** [review]

**Pull request merged**